src/components/Pages/ChartPage.js

```diff
@@ -18,6 +18,7 @@ export const ChartPage = ({ match }) => {
   const [sidebarData, setSidebarData] = useState(0);
   const [isSidebarOpen, setIsSidebarOpen] = useState(false);
 
+  // Toggle Sidebar to open and close
```

Comment:
this comment doesnt add that much value... the name toggleSidebar kind of explains what it is already